### ISSUE TITLE
Node.JS and Multithreading

### ISSUE THREAD

**Issue by sattha** [nodeschool/bangkok#9]
I'm one fan of Node.js when come to web application development (majority is Android Developer). Even non-blocking I/O is one core advantage of Node.JS, but I want to know whether Node.JS is  support Multithreading or not?  If not, will it be in the future ? Thank you.

**Comment by stefanmirck** [comment]
Hi @sattha, I’m an organizer for NodeSchool Amsterdam and while looking through my unread notifications (organizers are automatically subscribed to all NodeSchool repos) I came across your question.

Not too long ago I found a great post on Stack Overflow that (hopefully) answers your question: http://stackoverflow.com/questions/2387724/node-js-on-multi-core-machines/8685968#8685968.

TL;DR: no, Node.js doesn’t support multi-threading, but you can spawn child or worker processes.

**Comment by sattha** [comment]
Thank you for feedback.
